solver/llbsolver/ops/source.go

```diff
@@ -67,21 +67,27 @@ func (s *sourceOp) CacheMap(ctx context.Context, g session.Group, index int) (*s
 	if err != nil {
 		return nil, false, err
 	}
-	k, cacheOpts, done, err := src.CacheKey(ctx, g, index)
+
+	k, srcDigest, cacheOpts, done, err := src.CacheKey(ctx, g, index)
```

Comment:
I guess we can call `srcDigest` `pin` here for better readability.

Reply:
Sgtm